app/controllers/materials_controller.rb

```diff
@@ -0,0 +1,221 @@
+require 'rubygems'
```

Comment:
Also, making a Fat Model when you don't have any model you work on is rather hard.